### ISSUE TITLE
The dig method is undefined

### ISSUE THREAD

**Issue by pillarsdotnet** [jeremyevans/ruby-cicphash#2]
Added the following:
```
# Add a missing "dig" method to the CICPHash class.
class CICPHash 
  def dig(*keylist)
    val = self
    val = val.respond_to?(:[]) ? val[keylist.shift] : nil until keylist.empty?
  end
end
```

**Comment by jeremyevans** [comment]
I'm OK with adding this.  Is it possible for you to submit it as a pull request with tests?

**Comment by pillarsdotnet** [comment]
Working on it...

**Comment by jeremyevans** [comment]
Fixed by 173ce071aed58d06dbec9b2170379d7b65664dd9
